<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CopyFilesForm</class>
 <widget class="QWidget" name="CopyFilesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>397</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>459</width>
    <height>397</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>459</width>
    <height>397</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Настройка копирования файлов</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_1" stretch="0,0">
       <item>
        <widget class="QLineEdit" name="dir_line_edit_from">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="TypeField" stdset="0">
          <string>edit_from</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dir_button_dial_from">
         <property name="text">
          <string>Обзор</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
       <item>
        <widget class="QLineEdit" name="dir_line_edit_to">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="TypeField" stdset="0">
          <string>edit_to</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dir_button_dial_to">
         <property name="text">
          <string>Обзор</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0,0,0">
         <property name="spacing">
          <number>17</number>
         </property>
         <item>
          <widget class="QPushButton" name="add_files_name_button">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Добавить названия файлов</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delete_file_name_button">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Удалить файл</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="table_files">
         <property name="minimumSize">
          <size>
           <width>256</width>
           <height>305</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>256</width>
           <height>305</height>
          </size>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="columnCount">
          <number>2</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column/>
         <column/>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
